### ISSUE TITLE
amp-sidebar - empty div at bottom does not work correctly

### ISSUE THREAD

**Issue by fox94610** [ampproject/amphtml#22012]
The empty div injected to the bottom of the component to compensate for browser tray (unavoidable native UI element in both Chrome and Safari), does not push the sidebar content up, making scrolling to bottom sidebar content impossible without a developer adding a extra hack on top of the existing workaround.

Directly related to close issue:
"amp-sidebar - make the empty div configurable"
https://github.com/ampproject/amphtml/issues/14632
Related:
"Style amp-sidebar to make options accessible in Safari"
https://github.com/ampproject/amphtml/pull/16834

Screen shot in Safari on iPhone:
![iPhone-with-Safari](https://user-images.githubusercontent.com/243347/56823990-2cace200-680a-11e9-817a-3a195b048a69.jpg)

Screen shot in Chrome on iPhone:
![iPhone-with-Chrome](https://user-images.githubusercontent.com/243347/56824003-36364a00-680a-11e9-961d-da1094c02649.jpg)

Empty div shows up in Chrome Developer tools
![ss1](https://user-images.githubusercontent.com/243347/56824185-9b8a3b00-680a-11e9-90b5-a6cb7a86c3a7.jpg)

**Comment by aghassemi** [comment]
Thanks for the detailed report and researching existing issues.
